wiki/models/pluginbase.py

```diff
@@ -188,17 +197,21 @@ class RevisionPluginRevision(BaseRevisionMixin, models.Model):
     """
     If you want your plugin to maintain revisions, make an extra model
     that extends from this one.
-    
-    (this class is very much copied from wiki.models.article.ArticleRevision
+
+    (this class is very much copied from wiki.models.article.ArticleRevision)
+
+    .. pii: This model store IP the addresses of users who have edited the object
```

Comment:
Okay, since this syntax is used more than once, maybe I just don't understand what it means but other engineers would?